hayeon/programmers-0/소인수분해.js

```diff
@@ -0,0 +1,29 @@
+function solution(n) {
+  const result = [];
+
+  while (n % 2 === 0) {
+    result.push(2);
+    n = n / 2;
+  }
+
+  for (let i = 3; i * i <= n; i += 2) {
+    while (n % i === 0) {
+      result.push(i);
+      n = n / i;
+    }
+  }
+
+  if (n > 2) result.push(n);
+
+  return [...new Set(result)];
+}
```

Comment:
```suggestion
function solution(n) {
  const set = new Set();

  while (n % 2 === 0) {
    set.add(2);
    n /= 2;
  }

  for (let i = 3; i * i <= n; i += 2) {
    while (n % i === 0) {
      set.add(i);
      n /= i;
    }
  }

  if (n > 2) set.add(n);

  return [...set];
}
```
- 큰 문제는 아닌데 애초에 set을 선언하고 해도 좋을 것 같고, /= 연산자를 사용하는 것도 좋아보입니다 고생하셨씁니다